extra/foreman-tasks-cleanup.sh

```diff
@@ -119,6 +134,8 @@ while true; do
     shift
 done
 
+validate_options!
```

Comment:
I do not like the bang in the method name as it has special meaning in shell, tho it should work. I'm ok, just saying :)